plugins/host/main.c

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Open Mobile Platform LLC
+ */
+
+#include <err.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <unistd.h>
+
+/* OP-TEE TEE client API (built by optee_client) */
+#include <tee_client_api.h>
+
+/* For the UUID (found in the TA's h-file(s)) */
+#include <plugin_ta.h>
+
+#define SLEEP_SEC 2
+#define TA_PING_CNT 5
+
+int main(void)
+{
+	int i = 0;
+	TEEC_Result res = TEEC_SUCCESS;
+	TEEC_Context ctx = { };
+	TEEC_Session sess = { };
+	TEEC_Operation op = { };
+	TEEC_UUID uuid = PLUGIN_TA_UUID;
+	uint32_t err_origin = 0;
+
+	/* Initialize a context connecting us to the TEE */
+	res = TEEC_InitializeContext(NULL, &ctx);
+	if (res != TEEC_SUCCESS)
+		errx(1, "TEEC_InitializeContext failed with code 0x%x", res);
```

Comment:
Please use `%#"PRIx32` instead of `0x%x"`. `res` is a basically `uint32_t`.
Same below for the other format strings.